lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -248,23 +248,23 @@ <h3 class="hd hd-3">${_("Course Team Management")}</h3>
       data-add-button-label="${_("Add Moderator")}"
     ></div>
 
-    <div class="auth-list-container"
+    <div class="auth-list-container group_moderator"
       data-rolename="Group Moderator"
-      data-display-name="${_("Discussion Group Moderators")}"
+      data-display-name="${_("Discussion Group Community TA")}"
       data-info-text="
-        ${_("Discussion Group Moderators can edit or delete any post, clear misuse flags, close "
-         "and re-open threads, endorse responses, and see posts from all groups. "
-         "Their posts are marked as 'staff'. They cannot manage course team membership by "
-         "adding or removing discussion moderation roles. Only enrolled users can be "
-         "added as Discussion Moderators.")}"
+        ${_("Discussion Group Community TAs are members of the community whom you deem particularly "
```

Comment:
"Group Community TAs are members of the community who help course teams moderate discussions. Group Community TAs see only posts by learners in their assigned group. They can edit or delete posts, clear flags, close and re-open threads, and endorse responses, but only for posts by learners in their group."
(@staubina If they can be associated with more than one group, then "groups" plural, otherwise "group".)